### ISSUE TITLE
Flow types declared for non-existing redux-form.actions

### ISSUE THREAD

**Issue by ktosiek** [erikras/redux-form#3224]
### What is the current behavior?
Actions are exported as redux-form.${action}, but flow declarations say it's redux-form.actions.${action}. For example this should work, but flow check fails:

```
// @flow
import * as ReduxForm from 'redux-form'

ReduxForm.initialize('some-form', {}, {})
```

Flow error:
```
react/t.js:4
  4: ReduxForm.initialize('some-form', {}, {})
               ^^^^^^^^^^ property `initialize`. Property not found in
  4: ReduxForm.initialize('some-form', {}, {})
     ^^^^^^^^^ exports of "redux-form"
```

### What is the expected behavior?
Flow declarations should match code.

### What's your environment?
redux-form 7.0.1

**Comment by Pegase745** [comment]
Mentionned here #3214

**Comment by erikras** [comment]
Published fix in [`v7.0.2`](https://github.com/erikras/redux-form/releases/tag/v7.0.2).
